### PR TITLE
feat: upgrade swc and dprint-swc-ecma-ast-view

### PR DESCRIPTION
**Pull request by magurotuna**
This upgrade should fix https://github.com/denoland/deno_lint/issues/1011
See also https://github.com/dprint/dprint-swc-ecma-ast-view/pull/34

### CODE REVIEW

**Comment by dsherret** [comment]
@magurotuna thanks! I'm planning to test and do the full upgrade on Monday. We do a lot more testing across all crates before upgrading swc nowadays because we ran into major issues upgrading too many times https://github.com/denoland/deno_ast/tree/main/scripts (we'll need to upgrade those scripts because it needs to include https://github.com/denoland/eszip).

**Comment by magurotuna** [comment]
@dsherret Oh I didn't know that, thanks for the info. Sure, then I'll wait for that :) Should I close this PR in favor of your upgrading process?

**Comment by magurotuna** [comment]
@dsherret Alright, have a good night!

File: src/parsing.rs [hunk 1]
```diff
@@ -247,6 +247,7 @@ pub fn get_es_config(jsx: bool) -> EsConfig {
     import_assertions: true,
     static_blocks: true,
     private_in_object: true,
+    allow_super_outside_method: false,
```
[the remark this answers is not in the document]
Comment (magurotuna) [reply src/parsing.rs]: > We want to still be able to analyze code that has this and we can leave this for v8 to error if necessary.

Makes a lot of sense. It's now turned to `true` [654e896](https://github.com/denoland/deno_ast/pull/64/commits/654e89673cc1fadc0b1f31cf15975c20b54b64d6)

**Pull request merged**